common/djangoapps/entitlements/api/v1/tests/test_views.py

```diff
@@ -160,10 +203,27 @@ def test_get_entitlement_by_uuid(self):
         assert response.status_code == 200
 
         results = response.data
-        assert results == CourseEntitlementSerializer(entitlement).data
+        assert results == CourseEntitlementSerializer(entitlement).data and results.get('expired_at') is None
+
+    def test_get_expired_entitlement_by_uuid(self):
+        past_datetime = datetime.utcnow().replace(tzinfo=pytz.UTC) - timedelta(days=365 * 2)
+        entitlement = CourseEntitlementFactory(created=past_datetime)
+        CourseEntitlementFactory.create_batch(2)
+
+        CourseEntitlementFactory()
+        url = reverse(self.ENTITLEMENTS_DETAILS_PATH, args=[str(entitlement.uuid)])
+
+        response = self.client.get(
+            url,
+            content_type='application/json',
+        )
+        assert response.status_code == 200
+
+        results = response.data  # pylint: disable=no-member
+        assert results.get('expired_at')
```

Comment:
It might be good to asset that the entitlements were updated in the DB, and that we didn't just serialize and return an expired_at date.